frontend/src/App.css

```diff
@@ -55,3 +55,57 @@
   background:  #B91C1C!important;
   border-radius: 12px !important;
 }
+
+
+
+::-webkit-scrollbar {
+  width: 7px;
+}
+
+/* Track */
+::-webkit-scrollbar-track {
+  background: #fafafc; 
+}
+
+/* Handle */
+::-webkit-scrollbar-thumb {
+  background:  #B91C1C !important;
+  border-radius: 12px !important;
+}
+
+::-webkit-scrollbar-thumb:hover {
+  background:  #B91C1C!important;
+  border-radius: 12px !important;
+}
+
+
+
+::-webkit-scrollbar {
+  width: 7px;
+}
+
+/* Track */
+::-webkit-scrollbar-track {
+  background: #fafafc; 
+}
+
+/* Handle */
+::-webkit-scrollbar-thumb {
+  background:  #B91C1C !important;
+  border-radius: 12px !important;
+}
+
+::-webkit-scrollbar-thumb:hover {
+  background:  #B91C1C!important;
+  border-radius: 12px !important;
+}
+
+@media (max-width: 600px) {
+  ::-webkit-scrollbar {
+    width: 7px;
+  }
+
+  ::-webkit-scrollbar-thumb {
+    background: #B91C1C;
+  }
+}
```

Comment:
@VinayLodhi1712 why the styles here for the scroll bar is repeated ex: ::-webkit-scrollbar

Reply:
i can correct it i think forget pull the changes before pushing so it merge the all changes.